lib/build/bundle.js

```diff
@@ -5,6 +5,7 @@ const Convert = require('./convert-source-map');
 const fs = require('../file-system');
 const SourceInclusion = require('./source-inclusion').SourceInclusion;
 const DependencyInclusion = require('./dependency-inclusion').DependencyInclusion;
+const copyProperty = 'copy';
```

Comment:
There's nothing wrong in my opinion. The only reason for that is because you said "code it up in a way that makes renaming easy down the road" (we weren't sure about `copy` or `copyToDist`).  So, I thought this would make renaming easier. I vote for using `x.copy` directly